release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -642,6 +679,21 @@ submodule openconfig-bgp-common {
     }
   }
 
+  grouping bgp-common-use-multiple-paths-link-bandwidth-config {
+    description
+      "Parameters controlling usage of  of DMZ Link-Bandwidth
+      extended community in pultipath RIB/FIB formation";
+    leaf enable {
+      type boolean;
+      default "true";
+      description
+      "When set to TRUE, BGP multiplepath shall distributed traffic
```

Comment:
i'm assuming that when set to TRUE, wECMP only applies if `use-multiple-paths/config/enabled` and `use-multiple-paths/{i|e}bgp/config/maximum-paths` is > 1, otherwise setting this leaf to true has no effect?

In EOS, `ucmp mode 1` [1] has no effect unless `maximum-paths <n>` where n > 1 is also true

[1] https://www.arista.com/en/support/toi/eos-4-24-1f/14534-bgp-ucmp
```
‘ucmp mode 1 [<M> [<deviation%>]]’ is a mandatory command along with ‘maximum-paths <max-paths> [ecmp <max-ecmp>]’ command to enable UCMP for received routes at the DUT locally.
```